crates/sui-core/src/execution_driver.rs

```diff
@@ -103,15 +103,10 @@ pub async fn execution_process(
                     break;
                 }
             }
-
-            // Remove the certificate that finished execution from the pending_certificates table.
-            authority.certificate_executed(&digest, &epoch_store);
```

Comment:
why was this removed?

Reply:
I combined `certificate_executed()` and `objects_available()` into a single `notify_commit()` call from authority state, to simplify reasoning, so only 1 notification is needed after a transaction commits instead of 2. I kept `objects_available()` though to be called from `enqueue()` and tests.